src/main/java/com/cloudbees/jenkins/support/api/Content.java

```diff
@@ -36,14 +40,49 @@
 
     private final String name;
 
+    /**
+     * Parts of the name to be filtered
+     */
+    private String[] tokens;
```

Comment:
Might make more sense to call this `filterableParameters` or something like that. Calling this `tokens` is confusing since you're not really tokenizing anything.

Reply:
I initially called it something like that, but renamed it to something shorter. But as I'm a big fan of meaningful object names and it looks that you agree, I'll change them to your proposal. Thank you.